### PR TITLE
fix: page not loading after switch menu

### LINKED ISSUE #12: Component refresh exceptions
There seems to be a problem with the routing subscription mode, where components are not automatically updated when switching pages, but are rendered only after refreshing the interface

### PR DESCRIPTION
**Pull request by wu-clan**
please review.

### CODE REVIEW

File: src/layout/page-layout.vue [hunk 1]
```diff
@@ -16,8 +16,13 @@
 <script lang="ts" setup>
   import { computed } from 'vue';
   import { useTabBarStore } from '@/store';
+  import { useRoute } from 'vue-router';
 
   const tabBarStore = useTabBarStore();
+  const useRoure = useRoute();
+  const key = computed(() => {
+    return useRoure.path + Math.random();
```
Comment (downdawn) [inline src/layout/page-layout.vue]: Use random numbers? Isn't the path unique?

File: src/store/modules/app/index.ts [hunk 2]
```diff
@@ -3,30 +3,38 @@ import { Notification } from '@arco-design/web-vue';
...
-      title: menu.title,
+    const menuItem: AppRouteRecordRaw = {
+      path: !menu.path ? `/${convertToKebabCase(menu.name)}` : menu.path,
       name: menu.name,
-      path: !menu.path ? `/${localeName}` : menu.path,
       component: !menu.component
         ? () => import('@/views/not-found/index.vue')
         : views[`/src/views${menu.component}`],
       children: [],
+      fullPath: menu.path,
```
Comment (downdawn) [inline src/store/modules/app/index.ts]: fullPath should include the parent path?

**Review by downdawn — APPROVED** [review]

**Comment by downdawn** [comment]
LGTM.

File: src/store/modules/app/index.ts (hunk 2 above)
Comment (wu-clan) [reply src/store/modules/app/index.ts]: I think so.

File: src/layout/page-layout.vue (hunk 1 above)
Comment (wu-clan) [reply src/layout/page-layout.vue]: This is to force a component refresh every time you switch routes

**Pull request merged**